llvm/lib/CodeGen/LiveDebugValues/InstrRefBasedImpl.cpp

```diff
@@ -439,17 +454,20 @@ class TransferTracker {
     UseBeforeDefs.clear();
     UseBeforeDefVariables.clear();
 
-    // Map of the preferred location for each value.
-    DenseMap<ValueIDNum, LocationAndQuality> ValueToLoc;
+    // Mapping of the preferred locations for each value. Collected into this
+    // vector then sorted for easy searching.
+    SmallVector<ValueLocPair, 16> ValueToLoc;
```

Comment:
Any easy way to `reserve` here? And is smallsize of `16` more sensible then letting SmallVector choose? (I have no idea, just thinking out loud).